static/js/apps/insights/app.tsx

```diff
@@ -121,12 +127,16 @@ export function App(): JSX.Element {
         return;
       }
       let places = [place];
+      let topics = [topic];
       let cmpType = "";
       if (cmpPlaces && cmpPlaces.length > 0) {
         places = places.concat(cmpPlaces);
         cmpType = "ENTITY";
+      } else if (cmpTopic && cmpTopic !== undefined) {
```

Comment:
cmpTopic should be sufficient here? (If so, can change in later PR)

Reply:
Ok, will do...